device.go

```diff
@@ -24,11 +24,21 @@ import (
 
 // DeviceDesc is a representation of a USB device descriptor.
 type DeviceDesc struct {
-	// Bus information
-	Bus     int   // The bus on which the device was detected
-	Address int   // The address of the device on the bus
-	Speed   Speed // The negotiated operating speed for the device
-	Port    int   // The usb port on which the device was detected
+	// The bus on which the device was detected
+	Bus int
+	// The address of the device on the bus
+	Address int
+	// The negotiated operating speed for the device
+	Speed Speed
+	// The pyhsical port on the parent hub on which the device is connected. Ports are numbered from 1,
+	// excepting root hub devices which are always 0
+	Port int
+	// The physical path of connected parent port numbers, starting at the root hub device
+	// A path length of 1 indicates that this device is connected directly to a root hub,
+	// A path length of 2 or more are connected to intermediate devices,
+	// Root hub devices have an empty path
+	// i.e. [1,2,3] represents a device connected to port 3 of a device connected to port 2 of a device attached to port 1 of a root hub.
```

Comment:
reduce the line length to 80 chars here. A mix of short and long lines results in very confusing formatting in narrower views.

Reply:
[1,2,3] represents a device connected to port 3 of a **hub** connected to port 2 of a **hub** attached to port 1 of a root hub.